pkgs/applications/networking/gopher/geomyidae/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl, libressl,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "geomyidae";
+  version = "0.50.1";
+
+  src = fetchurl {
+    url = "gopher://bitreich.org/9/scm/geomyidae/tag/geomyidae-v0.50.1.tar.gz";
+    sha512 = "2a71b12f51c2ef8d6e791089f9eea49eb90a36be45b874d4234eba1e673186be945711be1f92508190f5c0a6f502f132c4b7cb82caf805a39a3f31903032ac47";
+  };
+
+  buildInputs = [ libressl ];
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace /usr/local $out
+    '';
```

Another option to avoid patching would be to override `PREFIX = /usr/local` using `make` arguments interface (untested): 

```suggestion
  makeFlags = [ "PREFIX=${placeholder "out"}" ];
```

---

Your suggestion works and is much nicer.